lib/puppet/settings/symlink_or_directory_setting.rb

```diff
@@ -0,0 +1,14 @@
+class Puppet::Settings::SymlinkOrDirectorySetting < Puppet::Settings::DirectorySetting
+
+  def initialize(args)
+    super
+  end
```

Comment:
Since this class doesn't change the behavior of the superclass' `initialize` method, you can drop this override.

Reply:
oh yeah, whoops, leftover copy pasta